packages/ui/components/profile/ProfileVoteButton.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  ChatAltIcon,
+  CheckIcon,
+  HandIcon,
+  XIcon,
+} from '@heroicons/react/outline'
+import clsx from 'clsx'
+import { useTranslation } from 'react-i18next'
+
+import { Button } from '../Button'
+
+export interface ProfileVoteButtonProps {
+  variant: 'abstain' | 'noWithVeto' | 'no' | 'yes'
+  pressed?: boolean
+}
+
+export const ProfileVoteButton = ({
+  variant,
+  pressed = false,
+}: ProfileVoteButtonProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <Button
+      className={clsx('pt-2 pb-6 pl-5 mb-2 w-full h-10', {
+        'border border-2 border-border-primary': pressed,
+      })}
```

Comment:
```suggestion
      className={clsx('pl-4 mb-2 w-full border-2 border-transparent', {
        'border-border-primary': pressed,
      })}
```

This will ensure the buttons don't change size when different ones are selected. No need to set height, it seems to just uncenter the children.